src/v0/util/index.js

```diff
@@ -1641,6 +1619,36 @@ const getEventReqMetadata = event => {
   return {};
 };
 
+/**
+ * error handler during transformation of a single rudder event for rt transform destinaiton
+ *
+ * This function is to be used only when we have a simple error handling scenario
+ * If some customisation wrt error handling is required, we recommend to not use this function
+ *
+ * @param {object} input - single rudder event
+ * @param {*} error - error occurred during transformation of a single rudder event
+ * @param {string} destType - destination name
+ * @returns
+ */
+const handleRtTfSingleEventError = (input, error, reqMetadata) => {
+  const errObj = generateErrorObject(error);
+
+  const resp = getErrorRespEvents(
+    [input.metadata],
+    errObj.status,
+    errObj.message,
+    errObj.statTags
+  );
+
+  errNotificationClient.notify(error, "Router Transformation (event level)", {
```

Comment:
Since, this is there, we need to remove error notification on Ln 1688.

Reply:
done